tsconfig.json

```diff
@@ -17,7 +17,7 @@
     "skipLibCheck": true,
     "sourceMap": true,
     "strictNullChecks": true,
-    "target": "es5",
+    "target": "es2015",
```

Comment:
- [ ] please check if in the tsconfig for the angular components, the target is set to es5

Reply:
Target is only set to "es5" in `libs/tools/builders/tsconfig.json` and `libs/tools/linting/tsconfig.json`. 
I think this should be fine.

Reply:
Please set the target in the `libs/barista-components/tsconfig.json` to `es5` thats the important part. This disables the usage of es6 features in the component library.